apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml

```
@@ -99,7 +99,7 @@ spec:
                     title: RPX-PR
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api|^HMCTS_Nightly.*CDM.*).+\/master
+                      ^HMCTS_(?!Nightly).*CDM.*\/(?!.*ccd-admin-web-api).+\/master
```

just be explicit? the regex there was to include nightly, if you don't want it just remove it
```suggestion
                      ^HMCTS_CDM.*\/(?!.*ccd-admin-web-api).+\/master
```